qemu/tests/fio_perf.py

```diff
@@ -209,7 +209,7 @@ def fio_install(tarball):
                 if s:
                     raise exceptions.TestFail("Failed to online disk: %s" % o)
     # format disk
-    if format:
+    if format == "True":
```

Comment:
I get param format from "format = params.get("format")", it returns a string, so it didn't work well previously.